crypt/api/crypt.script_api

```diff
@@ -84,3 +84,34 @@
     return:
       - name: output
         type: string
+
+
+#*****************************************************************************************************
+
+  - name: encrypt_xtea
```

Comment:
Perhaps should mention that the key should be <= 16 ?

Reply:
add 
desc: key should be <=16